pkg/aspect/root/flags/interceptor.go

```diff
@@ -102,10 +102,12 @@ func FlagsInterceptor(streams ioutils.Streams) interceptors.Interceptor {
 			// Search for config in home directory with name ".aspect" (without extension).
```

Comment:
comment now out-of-date?

Reply:
I am not fixing that in this PR. We will fix with #252.